lib/src/components/list_item/notification_list_item.dart

```diff
@@ -54,83 +62,115 @@ class ZetaNotificationListItem extends ZetaStatelessWidget {
       ..add(StringProperty('title', title))
       ..add(DiagnosticsProperty<bool>('notificationRead', notificationRead))
       ..add(DiagnosticsProperty<bool?>('showDivider', showDivider))
-      ..add(StringProperty('semanticLabel', semanticLabel));
+      ..add(StringProperty('semanticLabel', semanticLabel))
+      ..add(DiagnosticsProperty<Widget?>('attachment', attachment))
+      ..add(DiagnosticsProperty<bool?>('showBellIcon', showBellIcon));
   }
 
   @override
   Widget build(BuildContext context) {
     final colors = Zeta.of(context).colors;
-    return ZetaRoundedScope(
-      rounded: context.rounded,
-      child: Semantics(
-        explicitChildNodes: true,
-        label: semanticLabel,
-        button: true,
-        child: DecoratedBox(
-          decoration: _getStyle(context),
-          child: Column(
-            mainAxisSize: MainAxisSize.min,
-            mainAxisAlignment: MainAxisAlignment.spaceBetween,
-            children: [
-              Row(
-                mainAxisSize: MainAxisSize.min,
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: [
-                  leading,
-                  Expanded(
-                    child: Column(
-                      crossAxisAlignment: CrossAxisAlignment.start,
-                      children: [
-                        Row(
-                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                          children: [
-                            MergeSemantics(
-                              child: Row(
+    return Padding(
```

Comment:
Looks like there is one more feature to add - the button should be slidable:
See `ZetaChatItem` for an example.
![image](https://github.com/user-attachments/assets/3150d177-14c1-4c19-9845-492f132f0685)


https://www.figma.com/design/JesXQFLaPJLc1BdBM4sisI/%F0%9F%A6%93-ZDS---Components?node-id=30811-450&t=aBg4fvCW41ZQOnsL-4